contracts/core/RebalancingSetToken.sol

```diff
@@ -312,29 +357,29 @@ contract RebalancingSetToken is
         uint256[] memory outflowUnitArray = new uint256[](combinedTokenArray.length);
 
         // Get bid conversion price
-        uint256 priceNumerator = 1;
-        uint256 priceDivisor = 1;
+        uint256 priceNumerator = 1374;
```

Comment:
What is `1374`? Seems like a magic number.

Instead, make it a constant on the top of the file and ascribe a name to it.

Reply:
It's more a placeholder for when we put in an auction library. It will not be there in the final iteration.

Reply:
Ok, please leave a comment that this is a placeholder

Reply:
Omitting constants unless part of a library: https://ethereum.stackexchange.com/questions/13851/could-we-call-a-constant-function-without-spending-any-gas-inside-a-transaction/13855

Leave a comment for declarations like these